client/go/Taskfile.yml

```diff
@@ -65,6 +65,10 @@ tasks:
     desc: "Build the tun2socks binary and library for Linux"
     cmds: [{task: electron, vars: {TARGET_OS: "linux", TARGET_ARCH: "amd64"}}]
 
+  linux_arm64:
```

Comment:
Have you tried to build it from macOS, will this work?

Reply:
I don't have any macOS hw, so I can't check this